src/lib/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -1609,6 +1609,20 @@ public function insertRelation(RelationCreateStruct $createStruct): int
         return (int)$this->connection->lastInsertId(self::CONTENT_RELATION_SEQ);
     }
 
+    public function loadRelation(int $relationId): array
+    {
+        $query = $this->queryBuilder->createRelationFindQueryBuilder();
+        $expr = $query->expr();
+
+        $query
+            ->where(
```

Comment:
```suggestion
            ->andWhere(
```
should be used.

This method is preferred over `where`, because it adds conditions instead of replacing them. In case our query builder in it's `createRelationFindQueryBuilder()` method adds some "WHERE" clauses, those would be replaced by this call.